src/serac/numerics/odes.hpp

```diff
@@ -49,6 +49,8 @@ class SecondOrderODE : public mfem::SecondOrderTimeDependentOperator {
    * used by the residual operator
    */
   struct State {
+    double& time;
```

Comment:
doxygen comment required

Reply:
While I'm in there, I'll edit the doxygen comments on the other members that are misleading or incorrect.